### PR TITLE
Syslog: optionally disable legacy (clog) logging

### PR DESCRIPTION
**Pull request by AdSchellevis**
This should be all for https://github.com/opnsense/core/issues/4068, while there also ditched sshlockout_pf for a simpler script.  

See issue for background on this change.

### CODE REVIEW

**Comment by mimugmail** [comment]
Really nice, thank you! Businesses needs this for compliance. \o/

File: src/etc/inc/system.inc [hunk 1]
```diff
@@ -644,7 +644,6 @@ news.err;local0.none;local3.none;local4.none			{$log_directive}/var/log/system.l
 local7.none							{$log_directive}/var/log/system.log
 security.*							{$log_directive}/var/log/system.log
 auth.info;authpriv.info;daemon.info				{$log_directive}/var/log/system.log
-auth.info;authpriv.info;user.*					|exec /usr/local/sbin/sshlockout_pf 15
```
Comment (fichtner) [inline src/etc/inc/system.inc:647]: if we don't use it anymore please remove dependency from Makefile as well :)

File: src/opnsense/mvc/app/models/OPNsense/Core/ACL/ACL.xml [hunk 2]
```diff
@@ -312,7 +312,8 @@
     <page-diagnostics-logs-firewall-summary>
         <name>Diagnostics: Logs: Firewall: Summary View</name>
         <patterns>
-            <pattern>diag_logs_filter_summary.php*</pattern>
+          <pattern>ui/diagnostics/firewall/stats*</pattern>
```
Comment (fichtner) [inline src/opnsense/mvc/app/models/OPNsense/Core/ACL/ACL.xml:315]: indent is 2 here, usually 4 like before

File: src/etc/inc/system.inc (hunk 1 above)
Comment (AdSchellevis) [reply src/etc/inc/system.inc:647]: @fichtner will do :)

**Comment by AdSchellevis** [comment]
@fichtner I think it's safe to merge, any objections?

**Comment by fichtner** [comment]
Ps: plist?

**Pull request merged**